packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -248,9 +263,12 @@ class _DraggableState<T> extends State<DraggableBase<T>> {
       dragStartPoint: dragStartPoint,
       feedback: config.feedback,
       feedbackOffset: config.feedbackOffset,
-      onDragEnd: () {
+      onDragEnd: (Velocity velocity, bool wasAccepted) {
```

Comment:
Can you elaborate on how you'd use the velocity? Don't you need the point to be able to make use of it?